one_fm/api/tasks.py

```diff
@@ -260,12 +260,8 @@ def notify_checkin_checkout_final_reminder(recipients,log_type):
 					<a class="btn btn-primary" href="/app/shift-permission/new-shift-permission-1">Submit Shift Permission</a>&nbsp;
 					""")
 	notification_category = "Attendance"
-	checkout_subject = _("Final Reminder: Please checkout in the next five minutes.")
-	checkout_message = _("""
-		<a class="btn btn-danger" href="/app/face-recognition">Check Out</a>
-		Submit a Shift Permission if you are plannig to leave early or is there any issue in checkout or forget to checkout

```

Comment:
@SalShai The option Submit Shift Permission needs to be there as per the requirement in checkout reminder